demo/some-plugin.js

```diff
@@ -0,0 +1,57 @@
+const { BasePlugin, registerPlugin } = KalturaPlayer;
+
+import { AnotherIconComponent, AnotherPanelItemComponent, IconComponent, PanelItemComponent } from './components.js';
+
+export const pluginName = 'somePlugin';
+
+export class somePlugin extends BasePlugin {
+  static defaultConfig = {};
+
+  constructor(name, player) {
+    super(name, player);
+  }
+
+  loadMedia() {
+    const panelItemAId = this.player.getService('sidePanelsManager').addItem({
+      label: 'A',
+      expandMode: 'alongside',
+      renderIcon: IconComponent,
+      position: 'left',
+      presets: ['Playback', 'Live'],
+      renderContent: PanelItemComponent,
+    });
+
+    const PanelItemBId = this.player.getService('sidePanelsManager').addItem({
```

Comment:
add items in other panels as well. vertically and horizontally

Reply:
maybe log it to the console?

Reply:
fixed